### ISSUE TITLE
Submodule riscv-test-env URL is outdated.

### ISSUE THREAD

**Issue by thomascp** [cnrv/fpga-rocket-chip#21]
```
// fpga-rocket-chip/rocket-chip/torture
$ git diff
diff --git a/.gitmodules b/.gitmodules
index 953c66b..a557616 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -1,3 +1,3 @@
 [submodule "env"]
        path = env
-       url = git://github.com/ucb-bar/riscv-test-env.git
+       url = https://github.com/riscv/riscv-test-env.git
```

**Comment by wsong83** [comment]
I think I would simply accept it if it is put into a PR.

**Comment by thomascp** [comment]
It is in the `rocket-chip/torture/env`,  they already updated it [here](https://github.com/ucb-bar/riscv-torture/blob/99d8b2b441ecaa18b852505bb7718ee04e2753f5/.gitmodules), so I guess we can only explain this in the README.md, and remind the user to change this submodule URL?

```
$ cd rocket-chip/torture/
$ git submodule set-url env https://github.com/riscv/riscv-test-env.git
```

**Comment by wsong83** [comment]
We do not actually use torture. In this case, it might be easier for me just bump up the commit. Will do it in the following days. Thanks.

**Comment by wsong83** [comment]
Fixed by PR #22
